apps/src/music/Controls.jsx

```diff
@@ -6,14 +6,38 @@ import {Triggers} from '@cdo/apps/music/constants';
 import moduleStyles from './controls.module.scss';
 import BeatPad from './BeatPad';
 
-const Controls = ({isPlaying, setPlaying, playTrigger, top}) => {
-  const [isShowingBeatBad, setBeatPadShowing] = useState(false);
+const Controls = ({
+  isPlaying,
+  setPlaying,
+  playTrigger,
+  top,
+  startOverClicked
+}) => {
+  const [isShowingBeatPad, setBeatPadShowing] = useState(false);
   useEffect(() => {
     if (isPlaying) {
       setBeatPadShowing(true);
     }
   }, [isPlaying]);
 
+  const renderStartOver = () => {
+    return (
+      <button
+        type="button"
+        onClick={startOverClicked}
+        style={{
+          backgroundColor: 'rgba(0,0,0,0)',
+          color: 'white',
+          fontSize: 14,
+          padding: '0px 8px'
```

Comment:
nit: mind sticking this in the CSS file now that we have that set up?

Reply:
Good call.  Thanks for getting this going.